src/components/Charts/zoneUtils.js

```diff
@@ -0,0 +1,156 @@
+  { ...ZONE_MEDIUM, name: 'Stabilizing', value: 1.4 },
+  { ...ZONE_HIGH, name: 'Growing' },
+];
+
+const ZONES_POSITIVE_RATE = [
+  { ...ZONE_LOW, name: 'Low', value: 0.06 },
+  { ...ZONE_MEDIUM, name: 'Medium', value: 0.12 },
+  { ...ZONE_HIGH, name: 'High' },
+];
+
+const ZONES_HOSPITAL_USAGE = [
+  { ...ZONE_LOW, name: 'Sufficient', value: 0.25 },
+  { ...ZONE_MEDIUM, name: 'At risk', value: 0.8 },
+  { ...ZONE_HIGH, name: 'Overloaded' },
+];
+
+export const optionsRt = (data, endDate) => {
+  const { x, y } = last(data);
+  const [minYAxis, maxYAxis] = roundAxisLimits(0, getMaxY(data));
+  return {
+    ...baseOptions,
+    xAxis: {
+      ...baseOptions.xAxis,
+      max: parseDate(endDate),
+    },
+    yAxis: {
+      ...baseOptions.yAxis,
+      tickPositions: getTickPositions(minYAxis, maxYAxis, ZONES_RT),
+    },
+    series: [
+      {
+        className: 'ZoneChart__AreaRange',
+        type: 'arearange',
+        fillColor: '#efefef',
+        zIndex: 0,
+        enableMouseTracking: false,
+        data: data.map(d => [d.x, d.low, d.hi]),
+      },
+      {
+        name: 'Rt',
+        type: 'spline',
+        zones: ZONES_RT,
+        data,
+      },
+    ],
+    annotations: [
+      currentValueAnnotation(x, y, formatDecimal(y)),
+      ...zoneAnnotations(endDate, minYAxis, maxYAxis, y, ZONES_RT),
+    ],
+  };
+};
+
+export const optionsPositiveTests = (data, endDate) => {
+  const { x, y } = last(data);
+  const [minYAxis, maxYAxis] = roundAxisLimits(0, getMaxY(data));
+  return {
+    ...baseOptions,
+    xAxis: {
+      ...baseOptions.xAxis,
+      max: parseDate(endDate),
+    },
+    yAxis: {
+      ...baseOptions.yAxis,
+      labels: {
+        formatter: function () {
+          return formatPercent(this.value);
+        },
+      },
+      tickPositions: getTickPositions(minYAxis, maxYAxis, ZONES_POSITIVE_RATE),
+    },
+    annotations: [
+      currentValueAnnotation(x, y, formatPercent(y)),
+      ...zoneAnnotations(endDate, minYAxis, maxYAxis, y, ZONES_POSITIVE_RATE),
+    ],
+    series: [
+      {
+        name: 'Positive Tests',
+        data,
+        zones: ZONES_POSITIVE_RATE,
+      },
+    ],
+  };
+};
+
+export const optionsHospitalUsage = (data, endDate) => {
+  const { x, y } = last(data);
+  const [minYAxis, maxYAxis] = roundAxisLimits(0, getMaxY(data));
+  return {
+    ...baseOptions,
+    xAxis: {
+      ...baseOptions.xAxis,
+      max: parseDate(endDate),
+    },
+    yAxis: {
+      ...baseOptions.yAxis,
+      labels: {
+        formatter: function () {
+          return formatPercent(this.value);
+        },
+      },
+      tickPositions: getTickPositions(minYAxis, maxYAxis, ZONES_HOSPITAL_USAGE),
+    },
+    annotations: [
+      currentValueAnnotation(x, y, formatPercent(y)),
+      ...zoneAnnotations(endDate, minYAxis, maxYAxis, y, ZONES_HOSPITAL_USAGE),
+    ],
+    series: [
+      {
+        name: 'Hospital Usage',
+        data,
+        zones: ZONES_HOSPITAL_USAGE,
+      },
```

Comment:
this data is available as const rtData = projection.getDataset('rt'); will get this data now